packages/telemetry/index.cjs

```diff
@@ -1,8 +1,20 @@
 const secure = require('@lukeed/uuid/secure');
 const md5 = require('blueimp-md5');
-const Analytics = require('analytics-node');
 const { readJSONSync, writeJSONSync, pathExistsSync } = require('fs-extra');
 const wK = 'ydlp5unBbi75doGz89jC3P1Llb4QjYkM';
+const Analytics = require('analytics-node');
+
+/**
+ * @typedef {'usageStatsDisabled' |
+ * 			 'db-plugin-unvailable' |
+ * 			 'db-connection-error' |
+ * 			 'db-connection-success' |
+ * 			 'source-connector-not-found' |
```

Comment:
@hughess Added a new event `source-connector-not-found` for detecting when the connector is not found via build sources (I am guessing this happens pretty rarely). Can remove this if it's not that useful.